.github/workflows/Dev_User_CD.yml

```
@@ -7,30 +7,21 @@ on:
       - 'api-user/**'
       - 'domain/**'
+      - 'domain-redis/**'
       - 'common/**'
+      - 'common-test/**'
       - 'build.gradle'
       - 'settings.gradle'
       - '.github/workflows/Dev_User_CD.yml'
 
 jobs:
+  test:
+    uses: ./.github/workflows/Dev_CI.yml
```

actionlint (1.7.7)

error while parsing reusable workflow "./.github/workflows/Dev_CI.yml": "workflow_call" event trigger is not found in "on:" at line:4, column:3

(expression)